.rubocop.yml

```diff
@@ -1,13 +1,11 @@
 ---
-inherit_from: .rubocop_todo.yml
-
 require:
 - rubocop-performance
 - rubocop-rspec
 AllCops:
   NewCops: enable
   DisplayCopNames: true
-  TargetRubyVersion: '2.7'
+  TargetRubyVersion: '2.6'
```

Comment:
This should be `2.7`

Reply:
No, this is managed by the pdk template, which currently specifies `2.6`.

Reply:
I would suggest opening a PR to the pdk-template project if you feel strongly about it.